QUEST_LV_0100.tsv

```diff
@@ -5193,39 +5193,39 @@ QUEST_LV_0100_20150317_005192	It was a warning on the contaminated guardian. Def
+QUEST_LV_0100_20150317_005196	$Talk to Rolandas Jonas
 QUEST_LV_0100_20150317_005197	{memo X}
-QUEST_LV_0100_20150317_005198	Find the map in Vienisa Plateau
-QUEST_LV_0100_20150317_005199	Rolandas says the map for King Zachariel's Legacy is hidden where nobody knows. You have to check around because you don't know where it is exactly. First, check the southern area of Vienisa Plateau.
-QUEST_LV_0100_20150317_005200	Talk to Rolandas
-QUEST_LV_0100_20150317_005201	You can't possibly find it at once. It was fruitless. Go back to Rolandas and ask for the next area. 
-QUEST_LV_0100_20150317_005202	Check the area Rolandas Jonas told you
-QUEST_LV_0100_20150317_005203	Check the area Rolandas told you
-QUEST_LV_0100_20150317_005204	Search for the map in Radas Fork Road
-QUEST_LV_0100_20150317_005205	Rolandas says the map for King Zachariel's Legacy is hidden where nobody knows. You have to check around because you don't know where it is exactly. {nl}Search for the map in Radas Fork Road.
-QUEST_LV_0100_20150317_005206	Find the map in Rodoma Ruins 1
-QUEST_LV_0100_20150317_005207	Rolandas says the map for King Zachariel's Legacy is hidden where nobody knows. You have to check around because you don't know where it is exactly. Could it be found in Rodoma Ruins 1? 
-QUEST_LV_0100_20150317_005208	Search the area Rolandas told you
-QUEST_LV_0100_20150317_005209	Search for the map nearby grave manager's cabin
-QUEST_LV_0100_20150317_005210	Rolandas says the map for King Zachariel's Legacy is hidden where nobody knows. You have to check around because you don't know where it is exactly. Check around the cabin of grave manager.
-QUEST_LV_0100_20150317_005211	Find the map near the old cabin of grave manager.
-QUEST_LV_0100_20150317_005212	Look for the map around cabin of grave manager.
-QUEST_LV_0100_20150317_005213	Archaeologis Dezek is showing symptons of addiction to Scorpio's attacks. Talk to his assistant Irene.
-QUEST_LV_0100_20150317_005214	Find an antidote for the archaeologist
+QUEST_LV_0100_20150317_005198	$Search for the map in Vienisa Highland
+QUEST_LV_0100_20150317_005199	$Rolandas says the map of King Zachariel's inheritance is hidden where nobody knows. You have to check multiple areas to pinpoint its location. First, check the bottom area of Vienisa Highland.
+QUEST_LV_0100_20150317_005200	$Talk to Rolandas
+QUEST_LV_0100_20150317_005201	$You were unsuccessful on your first try. Go back to Rolandas and ask for the next area. 
+QUEST_LV_0100_20150317_005202	$Check the area Rolandas Jonas told you
+QUEST_LV_0100_20150317_005203	$Check the area Rolandas told you
+QUEST_LV_0100_20150317_005204	$Search for the map in Radas Fork Road
+QUEST_LV_0100_20150317_005205	$Rolandas says the map of King Zachariel's inheritance is hidden where nobody knows. You have to check multiple areas to pinpoint its location. {nl}Search for the map in Radas Fork Road.
+QUEST_LV_0100_20150317_005206	$Search for the map in Rodoma 1st Ruins
```

Comment:
wouldn't adding this make it seem like Zachariel is an inheritance?

I'll go with the "the map of the Great King Zachariel's inheritance"? is that ok?